lib/ecto_job/supervisor.ex

```diff
@@ -33,15 +33,15 @@ defmodule EctoJob.Supervisor do
   Starts an EctoJob queue supervisor
   """
   @spec start_link(Config.t) :: {:ok, pid}
-  def start_link(config = %Config{repo: repo, schema: schema, max_demand: max_demand, poll_interval: poll_interval}) do
+  def start_link(config = %Config{repo: repo, schema: schema, max_demand: max_demand, poll_interval: poll_interval, base_expiry_seconds: base_expiry_seconds}) do
     supervisor_name = String.to_atom("#{schema}.Supervisor")
     notifier_name = String.to_atom("#{schema}.Notifier")
     producer_name = String.to_atom("#{schema}.Producer")
 
     children = [
       worker(Postgrex.Notifications, [repo.config() ++ [name: notifier_name]]),
       worker(Producer, [
-        [name: producer_name, repo: repo, schema: schema, notifier: notifier_name, poll_interval: poll_interval]
+        [name: producer_name, repo: repo, schema: schema, notifier: notifier_name, poll_interval: poll_interval, base_expiry_seconds: base_expiry_seconds]
```

Comment:
I think it'd be good to run the code formatter to address the line length on some of these. Figured it could be a different PR and do the whole codebase.